internal/prompter/prompter.go

```diff
@@ -27,15 +30,232 @@ type Prompter interface {
 }
 
 func New(editorCmd string, stdin ghPrompter.FileReader, stdout ghPrompter.FileWriter, stderr ghPrompter.FileWriter) Prompter {
-	return &surveyPrompter{
-		prompter:  ghPrompter.New(stdin, stdout, stderr),
-		stdin:     stdin,
-		stdout:    stdout,
-		stderr:    stderr,
-		editorCmd: editorCmd,
+	accessiblePrompterValue := os.Getenv("GH_SCREENREADER_FRIENDLY")
+	switch accessiblePrompterValue {
+	case "", "false", "0":
```

Comment:
```suggestion
	case "", "false", "0", "no":
```

Reply:
I realize our AC might not have touched on this, but I want to encourage adding `"no"` as a non-truthy option as we do in [other places](https://github.com/search?q=org%3Acli+%22%5C%22no%5C%22%22&type=code)